gc/base/standard/ParallelGlobalGC.cpp

```diff
@@ -185,6 +185,34 @@ fixObject(OMR_VMThread *omrVMThread, MM_HeapRegionDescriptor *region, omrobjectp
 	}
 }
 
+struct HeapSizes{
+	uintptr_t freeBytes;
+	uintptr_t objectBytes;
+};
+
+static void
+clearFreeEntry(OMR_VMThread *omrVMThread, MM_HeapRegionDescriptor *region, omrobjectptr_t object, void *userData)
+{
+	MM_GCExtensionsBase *extensions = MM_GCExtensionsBase::getExtensions(omrVMThread->_vm);
+	MM_ParallelGlobalGC *collector = (MM_ParallelGlobalGC *)extensions->getGlobalCollector();
+
+	if (extensions->objectModel.isDeadObject(object)) {
+		if (extensions->objectModel.isSingleSlotDeadObject(object)) {
+			((HeapSizes *)userData)->freeBytes += extensions->objectModel.getSizeInBytesSingleSlotDeadObject(object);
+		} else {
+			/* We only clear both linked and unlinked free entries.
+			 */
+			OMRZeroMemory((void *)((uintptr_t)object + sizeof(MM_HeapLinkedFreeHeader)), extensions->objectModel.getSizeInBytesMultiSlotDeadObject(object) - sizeof(MM_HeapLinkedFreeHeader));
```

Comment:
I believe OMRZeroMemory() required start address / size to be aligned to 8 on 64-bit platform. There is implicit limitation introduced by assembly implementation, at least for Power.  So, either we can guarantee this alignment or use memset() instead. This is slow path any way.

Reply:
while the memory we clear is 8 bytes aligned (we skip a free entry header what is of size 8 or 16 bytes, and free entry is also 8 byte aligned to start with), let's play safe (to be future proof) and use memset

Reply:
Update with memset()